extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -178,6 +223,46 @@ export class AmpAutocomplete extends AMP.BaseElement {
     this.action_ = Services.actionServiceForDoc(this.element);
     this.viewport_ = Services.viewportForDoc(this.element);
 
+    const inputElements = childElementsByTag(this.element, 'input');
+    const textareaElements = childElementsByTag(this.element, 'textarea');
+    userAssert(
+      inputElements.length + textareaElements.length === 1,
+      `${TAG} should contain exactly one <input> or <textarea> child`
+    );
+    this.inputElement_ =
+      /** @type {!HTMLInputElement} */ (inputElements[0] ||
+      textareaElements[0]);
+    if (this.inputElement_.hasAttribute('type')) {
+      const inputType = this.inputElement_.getAttribute('type');
+      userAssert(
+        inputType === 'text' || inputType === 'search',
+        `${TAG} requires the "type" attribute to be "text" or "search" if present on <input>`
+      );
+    }
+
+    this.inline_ = this.element.hasAttribute('inline');
+    if (this.inline_) {
+      userAssert(
+        isExperimentOn(this.win, 'amp-autocomplete'),
+        `Experiment ${TAG} is not turned on for "inline" attr.`
+      );
+      this.trigger_ = this.element.getAttribute('inline');
+    }
```

Comment:
I think we can get rid of this conditional. `trigger_` can always set to the attribute value, which would be undefined if unset, which is fine. 

Then you can just include the inline check in the assertion.

```js
this.inline_ = this.element.hasAttribute('inline');
this.trigger_ = this.element.getAttribute('inline') || '';

userAssert(
  !this.inline_ || isExperimentOn(this.win, 'amp-autocomplete'),
  `Experiment ${TAG} is not turned on for "inline" attr.`
);
```